### PR TITLE
Test custom reviewdog

### PR DESCRIPTION
**Pull request by OnkarRuikar**
I've removed the readonly check from reviewdog source..

### CODE REVIEW

**Comment by yin1999** [comment]
Unfortunately, the triggered pr-review-lint-companion workflow would be from the main branch :(

File: files/zh-cn/games/index.md [hunk 1]
```diff
@@ -7,12 +7,16 @@ slug: Games
 
 游戏是最流行的计算机活动之一。新技术不断涌现，使得开发者有可能开发出更好、更强大、可以在任何符合标准的 web 浏览器运行的游戏。
 
-## 开发网页游戏
+## 开发网页游戏
```
Comment (OnkarRuikar) [inline files/zh-cn/games/index.md]:
```suggestion
## 开发网页游戏
```

File: files/zh-cn/games/index.md [hunk 2]
```diff
@@ -7,12 +7,16 @@ slug: Games
...
+## 开发网页游戏  
 
 欢迎来到 MDN 游戏开发中心！在网站的这一模块，我们为想要开发游戏的 web 开发者提供了资源。你可以在左侧主菜单中找到很多有用的教程和技术文档，尽情去探索吧。
 
 我们同样提供了参考部分，因此你可以轻易地找到有关游戏开发所有常用的 API。
 
+```css
+div{backgorund-color: red;}
+```
+
```
Comment (OnkarRuikar) [inline files/zh-cn/games/index.md]:
```suggestion
```

**Comment by OnkarRuikar** [comment]
I've removed the changes from the md file. Could you commit this PR? Then I'll create a separate PR updating the md file to test this.

**Pull request merged**